README.md

```diff
@@ -1,2 +1,246 @@
+
+### Production Workload: AWS EKS
+
+For production workload development, you will want to use a cloud platform. 
+
+EKS is a managed Kubernetes platform provided by AWS. The eksctl tool is a simple way to bring up a Kubernetes cluster on EKS.
+
+#### Install eksctl
+
+To install eksctl, follow the steps described here: https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html
+
+#### Using eksctl to provision Kubernetes cluster on EKS
+
+The following code using eksctl can be used to create a Kubernetes cluster in EKS. You will need to replace CLUSTER_NAME, KUBERNETES_VERSION, REGION, NODEGROUP_NAME, NODE_TYPE and NUMBER_OF_NODES based on your configuration.
+
+```
+eksctl create cluster \
+  --name CLUSTER_NANE \
+  --version KUBERNETES_VERSION \
+  --region REGION \
+  --nodegroup-name NODEGROUP_NAME \
+  --node-type NODE_TYPE \
+  --nodes NUMBER_OF_NODES
+```
+
+#### Suggestions for Naming
+
+CLUSTER_NAME: Choose a distinctive cluster name.
+KUBERNETES_VERSION: For now, we only support the latest version of Kubernetes in EKS, which is 1.21.
+NODEGROUP_NAME: Choose a distinctive node group name.
+NODE_TYPE: The recommendation from our performance team is to use the r5.large node type for development purpose.
+NUMBER_OF_NODES: Total number of nodes running not only Marklogic database, but also nodes running other applications.
+
+# Install Marklogic Helm Chart
+
+## Add Marklogic Repo
+
+If you haven’t already, add the Marklogic official repo to Helm.
+
+```
+helm repo add marklogic https://github.com/marklogic/marklogic-kubernetes
+```
+
+The output will look like this:
+
+```
+"marklogic" has been added to your repositories
+```
+
+Use this command to verify that the repo has been added to Helm:
+
+```
+helm repo list
+```
+
+You should see an entry like this:
+
+`marklogic           https://github.com/marklogic/marklogic-kubernetes`
+
+Use this command to ensure the Helm repo is up to date:
+
+```
+helm repo update
+```
+
+## Installing the Chart
+
+Use this command to install marklogic chart to current namespace with default settings:
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1
+```
+
+After you install, the output would look like this:
+```
+NAME: my-release
+LAST DEPLOYED: 
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+```
+
+**Note:** --version=1.0.0-ea1 must be provided. You can choose a distinctive release name to replace "my-release"
+
+It's strongly recommend to deploy marklogic chart in exclusive namespace, use --create-namespace flag if the namespace have not be created:
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1 --namespace=marklogic --create-namespace
+```
+
+Use this command to verify the deployment:
+
+```
+helm list --all-namespaces
+```
+
+You should see an entry named "my-release" (or the release name you chose) with a status of "deployed".
+
+## Configuration Options for Installation
```

Comment:
Are these options just for installation? Should the title just be Configuration Options?